src/Application.vala

```diff
@@ -203,7 +203,6 @@ public class AppCenter.App : Gtk.Application {
         base.dbus_register (connection, object_path);
 
         if (silent) {
-            DBusServer.init ();
```

Comment:
The only thing in this method is a synchronous loading of all the installed applications. As far as I can tell, this is just so we don't get errors when we try to uninstall something that isn't installed with the DBus server. I've moved the installed check to just before uninstalling an app to prevent this.